homeassistant/components/airtouch4/__init__.py

```diff
@@ -0,0 +1,104 @@
+"""The AirTouch4 integration."""
+import logging
+
+from airtouch4pyapi import AirTouch
+
+from homeassistant.components.climate import SCAN_INTERVAL
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["climate"]
```

Comment:
Not used. Best would be to have `async_forward_entry_setup` below loop over this list.